src/css/_icons.scss

```diff
@@ -0,0 +1,29 @@
+// CSS styles for SVG icons used throughout video.js.
+//
+// The goal is to replace all icons from the font family pulled from videojs/font entirely.
+// This project currently uses fonts. We want to replace this with SVGs from
+// images/icons.svg. This will ensure consitency between versions, as well as simplified
+// and straight-forward customization.
+
+// Default styling for all SVG icons
+.vjs-svg-icon {
+  display: inline-block !important;
```

Comment:
Is the `!important` necessary here?

Reply:
No it is not! I was using important to overwrite the button css for icons, but I just added some new styling in that file instead. https://github.com/videojs/video.js/pull/8260/commits/357adf1b267515534e4253af49a46a59777fdc1d#diff-db2130389e768f54f29c1ef67a128d854ea929e98f23d561de7f5b534747cd91R31